docs/modules/ROOT/pages/node_pattern.adoc

```diff
@@ -409,6 +409,43 @@ has_user_data?(node, /^pass(word)?$/i)
 has_user_data?(node, ->(key) { # return true or false depending on key })
 ----
 
+== `%keyword` for named parameters
```

Comment:
Not sure if `keyword` is the best term in this context. Maybe something like `named_capture` or `param_name`/`name` would be better? Not sure.

Reply:
`%param_name` is good :+1: